.github/dependabot.yml

```
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: bundler
+  directory: "/"
+  schedule:
+    interval: daily
```

```suggestion
    interval: weekly
```
I don't think this repo needs daily dependency bumps